### DISCUSSION TITLE
import account using private key

### DISCUSSION THREAD

**Discussion by billtlee** [Web3Auth/Web3Auth#711] [Help]
Hi,

Is there an example of how we can import an account using private key?  Like for example, allowing the user to login to our testnet and our production with the same wallet.  Currently, by default when a new user logs in using their social login a new wallet is created.

Thanks!

**Comment by himanshuchawla009** [comment]
No, that is not possible and not suggested as well. Since testnet is a testing network  so there are high chances of importing compromised account, so we don't suggest/have feature to import external keys or testnet keys in web3auth mainnet.

**Comment by billtlee** [comment]
@himanshuchawla009 Thanks for the reply.  The reason for doing this is for us to do airdrop to users from our testnet.  What would you suggest as a way for us to airdrop to testnet users?  Thanks!

**Reply by himanshuchawla009** [reply]
in this case testnet is web3auth testnet or your chain testnet?

**Comment by billtlee** [comment]
Our testnet is on Goerli using web3auth and want to do airdrop when we move to mainnet.

**Reply by himanshuchawla009** [reply]
account will remain same in that case, just make sure to set web3auth network as mainnet in web3auth constructor, keep same clientId and verifier (in case you are using custom verifier) in web3auth constructor params

**Comment by billtlee** [comment]
We are using Moralis and currently have 2 instance of testnet, a nightly and an alpha testnet.  Both are on Goreli.  When I use my Google account to login, it's creating a new wallet for me.  Maybe it's the settings from their adapter?  We are using their v1 adaptor.  I think they are now using the web3auth wagmi connector.  If we move to the new wagmi, how do I maintain the same wallet address across environments?  Thanks!

**Reply by himanshuchawla009** [reply]
are you passing same clientIds in both instances?

**Comment by billtlee** [comment]
I am using 2 clientid's.  Is that the issue?  So, if I use the same client id's, the wallet address will be the same I assume?

**Reply by YZhenY** [reply]
yes

**Comment by billtlee** [comment]
Thanks!
